contents/docs/new-to-posthog/revenue.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Capturing revenue data with PostHog"
+---
+
+import { AnalyticsPlannerTip } from 'components/Docs/AnalyticsPlannerTip'
+import { Admonition } from 'components/Docs/Admonition'
+
+<AnalyticsPlannerTip />
+
+While the analytics tasks so far have been full of subtlety, revenue is easy.
+
+It's binary.
+
+Either someone has given us money or they have not.
+
+# Basic revenue capture
```

Comment:
If you are using Stripe or Chargebee, you don't even need to care what revenue capture looks like, you can just connect your source and query it. 

The real tricky part is what that query looks like. As far as I know, we can get some basic revenue metrics, but more complex ones like MRR are still not easy to set up.